monitoring/base/rhods-rules.yaml

```diff
@@ -16,5 +16,9 @@ spec:
       expr: count(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}==1)
     - record: cluster:usage:consumption:rhods:active_users
       expr: count(count_over_time(kube_pod_container_status_ready{namespace="rhods-notebooks",pod=~"jupyterhub-nb.*"}[1h]))
+    - record: cluster:usage:consumption:rhods:cpu_requests_runtime
+        expr: sum(kube_pod_container_resource_requests{namespace='rhods-notebooks', resource='cpu'})
+    - record: cluster:usage:consumption:rhods:cpu_limits_runtime
+        expr: sum(kube_pod_container_resource_limits{namespace='rhods-notebooks', resource='cpu'})
       labels:
```

Comment:
I think your changes should be placed _after_ this label

Reply:
Done, updated to have the label moved up to the `cluster:usage:consumption:rhods:active_users` rule based on discussion with @anishasthana.